openapi.properties

```diff
@@ -236,7 +236,7 @@ openapi/soundcloud=f05addeb0b7344907449f4ba15c2822b
 openapi/spotify=49dd1df3dd1cac9466d5295d397a6bfb
 openapi/spotto=498ae2dc74d589df9320b16ef6b847df
 openapi/storecove=734a1f409ad4b9ef84ad81f5d42fd9f1
-openapi/stripe=04ef831b0e9b3bb256be6d8451aac64c
+openapi/stripe=b0effb32175e8df35f5df98f08125ad0
```

Comment:
Recently we did some changes related to icon via https://github.com/ballerina-platform/openapi-connectors/pull/562

So stripe has to be released in the next release workflow.

I think this hash update will block the release of stripe connector.

Shall we not commit this hash? WDYT?

Reply:
Updated the PR removing the hash change.